server/start.go

```diff
@@ -244,6 +247,53 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 	return WaitForQuitSignals()
 }
 
+func restoreLocalSnapshot(ctx *Context, app types.Application, ssRestoreHeight uint64) error {
+	ctx.Logger.Info(fmt.Sprintf("Local Statesync Restore Snapshot With Height: %d", ssRestoreHeight))
+	ctx.Config.StateSync.RestoreHeight = ssRestoreHeight // Should be a one-shot command
+
+	ctx.Logger.Info("Searching local snapshots")
+	resp := app.ListSnapshots(abci.RequestListSnapshots{})
+	if len(resp.Snapshots) < 1 {
+		panic("No available snapshots")
```

Comment:
We should not be invoking panics when this function returns an error.
```suggestion
		return fmt.Errorf("...")
```